cargo-quickinstall/src/main.rs

```diff
@@ -34,11 +37,22 @@ fn get_latest_version(crate_name: &str) -> std::io::Result<String> {
             --user-agent 'cargo-quickinstall build pipeline (alsuren@gmail.com)' \
             --location \
             --fail \
-            'https://crates.io/api/v1/crates/{}' \
-            | jq -r .versions[0].num",
+            'https://crates.io/api/v1/crates/{}'",
         crate_name
     );
-    bash_stdout(&command_string)
+    match bash_stdout(&command_string) {
+        Ok(stdout) => match stdout.parse() {
+            Ok(parsed) => {
+                let parsed: JsonValue = parsed;
+                let mut version: String = parsed["versions"][0]["num"].stringify().unwrap();
+                version.remove(0);
+                version.remove(version.len()-1);
```

Comment:
There might be a way to avoid needing to strip off the `"` at the start and end here.

Reply:
Ok, let me check!

Reply:
`tinyjson`'s `stringify()` method returns the string with "" intentionally. We could create a function that does exactly the same thing but without the "". What do you think?